chain-signatures/contract/src/lib.rs

```diff
@@ -612,7 +603,7 @@ impl VersionedMpcContract {
 
     // This function can be used to transfer the MPC network to a new contract.
     #[private]
-    #[init(ignore_state)]
```

Comment:
This function is less important since we plan to stick to 1 contract.
If we decide to move to another one - it should not have any state.